dev-libs/date/date-3.0.3.ebuild

```diff
@@ -31,5 +29,5 @@ src_configure() {
 
 src_test() {
 	cd "${SRC_DIR}"test/ || die
-	./testit || die
+	CC="$(tc-getCC)" ./testit || die
```

Comment:
It's written in C++ though.

Reply:
Fixed in be39248e48dbe1e168ac79144178b7250e99c6ec.

Reply:
(Not dropped CC; please check carefully whether it builds anything with CC first. Perhaps by doing CC=false.)